src/pimlify-vue/src/router.ts

```diff
@@ -1,24 +1,32 @@
-import OrderMenu from './components/OrderMenu.vue';
-import RestaurantComponent from './components/RestaurantComponent.vue';
-import RegisterComponent from './components/RegisterComponent.vue';
+import OrderMenu from "./components/OrderMenu.vue";
```

Comment:
We should define single or double quotes. I would prefer single 👍

Reply:
do that ;) provide an prettierrc or i don't know... check in the workspace settings